source/core/response.ts

```diff
@@ -91,6 +91,11 @@ export interface PlainResponse extends IncomingMessageWithTimings {
 	The result of the request.
 	*/
 	body?: unknown;
+
+	/**
+	Whether the response status code is 2xx.
```

Comment:
```suggestion
	Whether the response was successful (status code in the range 200-299).
```

Reply:
Should also mention that it's not necessary to check this property unless `{throwHttpErrors: false}`.